packages/themes/src/themes/console/dark/index.ts

```diff
@@ -0,0 +1,8 @@
+import deepmerge from 'deepmerge'
+import { deprecatedDarkTokens } from '../deprecated/dark'
+import { darkTheme } from './__generated__'
+
+export const consoleDarkTheme = deepmerge(
+  darkTheme,
+  deprecatedDarkTokens,
+) as typeof darkTheme & typeof deprecatedDarkTokens
```

Comment:
Would it work to just type: `export const consoleDarkTheme: typeof darkTheme & typeof deprecatedDarkTokens` ?